sample/mpp-library/src/commonMain/resources/MR/colors/colors.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="textColor">EE2050FF</color>
+    <color name="whiteThemed">
+        <light>FFFFFFFF</light>
+        <dark>000000FF</dark>
+    </color>
```

Comment:
i think we should test this in sample app

Reply:
also here RGBA color. but in code you read it as ARGB...

Reply:
Now all colors parses as RGBA.